source/braille.py

```diff
@@ -348,6 +349,9 @@
 #: braille displays should be automatically detected and used.
 #: @type: str
 AUTO_DISPLAY_NAME = AUTOMATIC_PORT[0]
+#: The name of the noBraille display driver
+#: @type: str
+NO_BRAILLE_DISPLAY_NAME = "noBraille"

```

Comment:
I think new lines make this more readable.
The type hint is unnecessary

```suggestion

#: The name of the noBraille display driver
NO_BRAILLE_DISPLAY_NAME: str = "noBraille"

```